vividus-extension-selenium/src/main/java/org/vividus/selenium/screenshot/AbstractScreenshotTaker.java

```diff
@@ -97,6 +87,15 @@ public Path takeScreenshot(Path screenshotFilePath) throws IOException
         return null;
     }
 
+    @Override
+    public Optional<Screenshot> takeScreenshot(String screenshotName)
+    {
+        byte[] screenshotData = takeScreenshotAsByteArray();
+        return screenshotData.length > 0
+                ? Optional.of(new Screenshot(generateScreenshotFileName(screenshotName), screenshotData))
+                : Optional.empty();
+    }
```

Comment:
what's the reason to add a default implementation? Web and Mobile App screenshot takers have this method implemented, but in different ways